crates/solver/src/liquidity.rs

```diff
@@ -91,6 +91,7 @@ pub struct LimitOrder {
     /// factor to make matching orders more valuable from an objective value
     /// perspective.
     pub scaled_unsubsidized_fee: U256,
+    pub actual_fee_amount: U256,
```

Comment:
Is this different from unscaled_subsidized_fee above?

Reply:
I am very sorry. Either, I should not do things on the fly or I should take holiday.

Reply:
No worries. That's why we have reviews.